pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.38.0-dev50"
+__version__ = "0.38.0-rc10"
```

Comment:
```suggestion
__version__ = "0.38.0-rc11"
```
We will need to update the rec version.

Reply:
Because of PYPI.